src/main/java/com/jprotractor/NgNavigation.java

```diff
@@ -1,44 +1,55 @@
 package com.jprotractor;
 
 import java.net.URL;
-import org.openqa.selenium.WebDriver.Navigation;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebDriverException;
 
-public class NgNavigation implements Navigation {
-    @SuppressWarnings("unused")
-    private NgWebDriver NgDriver;
-    private Navigation Nav;
-    public Navigation WrappedNavigation;
-
-    public NgNavigation(NgWebDriver ngDriver, Navigation navigation) {
-        this.NgDriver = ngDriver;
-        this.Nav = navigation;
-        this.WrappedNavigation = this.Nav;
+/**
+ * Angular Navigation.
+ * @author Carlos Alexandro Becker (caarlos0@gmail.com)
```

Comment:
@sergueik Yeah, you can add your name here later, it's just that you're not adding it anywhere.

also, qulice will complain if no author is provided once it is fully enabled.